RecFCCeeCalorimeter/src/components/CaloTopoClusterFCCee.cpp

```diff
@@ -69,11 +72,16 @@ StatusCode CaloTopoClusterFCCee::initialize() {
     error() << "Unable to retrieve ECal Barrel cell positions tool!!!" << endmsg;
     return StatusCode::FAILURE;
   }
-  // Check if cell position HCal Barrel tool available
-  if (!m_cellPositionsHCalBarrelTool.retrieve()) {
-    error() << "Unable to retrieve HCal Barrel cell positions tool!!!" << endmsg;
+  // Check if cell position HCal Barrel tool available (if name is set)
+  if (!m_cellPositionsHCalBarrelTool.empty()) {
```

Comment:
This condition seems to be always true, can you provide empty tool handle in the corresponding `declareProperty` above?

Reply:
fixed in latest commit